ros/api/v1/recommendation_ratings.py

```diff
@@ -5,46 +5,90 @@
 from ros.lib.utils import identity, user_data_from_identity
 
 
-class RecommendationRatingsApi(Resource):
+def validate_rating_post_api(func):
```

Comment:
Shall we move this decorator to utils as of now?
If we come up with more later, we can create a dedicated file.